python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -227,10 +227,9 @@ def _update(self):
 
         # Launch additional nodes of the default type, if still needed.
         num_workers = len(nodes) + num_pending
-        if num_workers < target_workers:
-            max_allowed = min(self.max_launch_batch,
-                              self.max_concurrent_launches - num_pending)
-
+        max_allowed = min(self.max_launch_batch,
```

Comment:
This is technically a minor bug fix (we were submitting launches with count -1 before this)